src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -2200,6 +2200,37 @@ private function addMailerSection(ArrayNodeDefinition $rootNode, callable $enabl
                             ->end()
                         ->end()
                     ->end()
+                    ->children()
+                        ->arrayNode('dkim_signer')
+                            ->fixXmlConfig('option')
+                            ->canBeEnabled()
+                            ->info('DKIM signer configuration')
+                            ->children()
+                                ->scalarNode('key')->end()
+                                ->scalarNode('domain')->end()
+                                ->scalarNode('select')->end()
+                                ->scalarNode('passphrase')->end()
+                                ->arrayNode('options')
+                                    ->performNoDeepMerging()
+                                    ->normalizeKeys(false)
+                                    ->useAttributeAsKey('name')
+                                    ->defaultValue([])
+                                    ->prototype('variable')->end()
+                                ->end()
+                            ->end()
+                        ->end()
+                        ->arrayNode('smime_signer')
+                            ->canBeEnabled()
+                            ->info('SMIME signer configuration')
+                            ->children()
+                                ->scalarNode('key')->end()
+                                ->scalarNode('certificate')->end()
```

Comment:
Those nodes about certificate should probably have an info explaining that they expects *paths*.